projects/plugins/crm/src/automation/class-automation-engine.php

```diff
@@ -8,32 +8,42 @@
 namespace Automattic\Jetpack\CRM\Automation;
 
 /**
- * Automation Engine
+ * Automation Engine.
  *
- * @package Automattic\Jetpack\CRM\Automation
+ * @since $$next-version$$
  */
 class Automation_Engine {
 
-	/** @var Automation_Engine Instance singleton */
+	/**
+	 * @var Automation_Engine Instance singleton.
+	 */
```

Comment:
This is the format shown [in the WP CS](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#2-1-class-members):

```php
/**
 * Summary.
 *
 * @since x.x.x
 * @var type $var Description.
 */
```

```suggestion
	/**
	 * A singleton instance of the engine.
	 *
	 * @since $$next-version$$
	 * @var Automation_Engine
	 */
```